app/src/features/apiClient/screens/PostmanImporterView/components/PostmanImporter/utils.ts

```diff
@@ -80,9 +80,9 @@ const processScripts = (item: any) => {
 
   item.event.forEach((event: any) => {
     if (event.listen === "prerequest") {
-      scripts.preRequest = migratePostmanScripts(event.script.exec.join("\n"));
+      scripts.preRequest = migratePostmanScripts(event.script?.exec?.join("\n"));
     } else if (event.listen === "test") {
-      scripts.postResponse = migratePostmanScripts(event.script.exec.join("\n"));
+      scripts.postResponse = migratePostmanScripts(event.script?.exec?.join("\n"));
```

Comment:
Instead of optional chaining we can have explicit checks. Process only if the script is defined